monadic-rx/shared/src/test/scala/mhtml/RxTests.scala

```diff
@@ -199,6 +199,32 @@ class RxTests extends FunSuite {
     assert(rx1.isCold && rx2.isCold)
   }
 
+  test("mergeN") {
+    // Should be identical to merge test above
+    val rx1: Var[Int] = Var(0)
+    val rx2: Var[Int] = Var(1)
+    val rxs = List(rx1, rx2)
+    val merged: Rx[Int] = rxs.merge
```

Comment:
It seems the code from this point in the test can easily be factored out, so assuming we want to do this, I will plan to use a common function from `test("merge")` and `test("mergeN")`

Reply:
Also, would need to add at least some doc to the README.

Reply:
I wouldn't factor out the tests. I think it a nice feature to have each test as a small isolated example, they are simpler to read and write :) (and should have zero maintenance cost, assuming backwards compatibility)

Reply:
OK. I guess I just feel that code duplication, even in tests, can be tricky. If a person is looking at one test, but forgets another test that is supposed to be the same, then over time they might begin to evolve into different organisms ;).